lib/bin/repl.js throws error in prod

I've tried out lib/bin/repl.js locally, which @alxndrsn wrote, and I've already found it to be a useful tool. However, I'm encountering an error when I try to run it in prod:

```
# docker compose exec service node lib/bin/repl.js
node:internal/modules/cjs/loader:1228
  throw err;
  ^

Error: Cannot find module 'should'
Require stack:
- /usr/odk/lib/bin/repl.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Module._load (node:internal/modules/cjs/loader:1051:27)
    at Module.require (node:internal/modules/cjs/loader:1311:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (/usr/odk/lib/bin/repl.js:10:16)
    at Module._compile (node:internal/modules/cjs/loader:1469:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)
    at Module.load (node:internal/modules/cjs/loader:1288:32)
    at Module._load (node:internal/modules/cjs/loader:1104:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/usr/odk/lib/bin/repl.js' ]
}
```

I think it's because `should` is a dev dependency. Could we just remove `should` from `repl.context` in lib/bin/repl.js? It might be a little less convenient locally, but it'd still be possible to access `should` locally: you can run `require('should')` after starting the REPL. It'd definitely be useful to be able to run lib/bin/repl.js in prod.

--- a/lib/bin/repl.js
+++ b/lib/bin/repl.js
@@ -20,7 +20,7 @@ const container = require('../util/default-container');
     context.should = require('should'); // eslint-disable-line import/no-extraneous-dependencies
     contextKeys.push('should'); // manually add due to should.js prototype pollution
     require('../../test/assertions');
-  } catch(err) { /* skip devDependencies if unavailable */ }
+  } catch (err) { /* skip devDependencies if unavailable */ }
 
   console.log('Available vars:', contextKeys.sort().join(', '));
 

--- a/lib/bin/repl.js
+++ b/lib/bin/repl.js
@@ -14,15 +14,16 @@ const container = require('../util/default-container');
 
 (async () => {
   const context = { ..._.omit(container, 'with'), container, sql };
-  const contextKeys = Object.keys(context);
+  const replGlobals = Object.keys(context);
 
   try {
-    context.should = require('should'); // eslint-disable-line import/no-extraneous-dependencies
-    contextKeys.push('should'); // manually add due to should.js prototype pollution
+    // should adds itself directly to global scope, so does not need to be added to the context
+    require('should'); // eslint-disable-line import/no-extraneous-dependencies
     require('../../test/assertions');
+    replGlobals.push('should');
   } catch (err) { /* skip devDependencies if unavailable */ }
 
-  console.log('Available vars:', contextKeys.sort().join(', '));
+  console.log('Available vars:', replGlobals.sort().join(', '));
 
   const repl = require('repl').start({
     useGlobal: true, // enable should.js prototype pollution

--- a/lib/bin/repl.js
+++ b/lib/bin/repl.js
@@ -7,18 +7,22 @@
 // including this file, may be copied, modified, propagated, or distributed
 // except according to the terms contained in the LICENSE file.
 
-const should = require('should'); // eslint-disable-line import/no-extraneous-dependencies
-require('../../test/assertions');
-
 const _ = require('lodash'); // eslint-disable-line import/no-extraneous-dependencies
 const { sql } = require('slonik');
 
 const container = require('../util/default-container');
 
 (async () => {
-  const context = { ..._.omit(container, 'with'), container, should, sql };
-  const contextKeys = Object.keys(context).sort();
-  console.log('Available vars:', contextKeys.join(', '));
+  const context = { ..._.omit(container, 'with'), container, sql };
+  const contextKeys = Object.keys(context);
+
+  try {
+    context.should = require('should'); // eslint-disable-line import/no-extraneous-dependencies
+    contextKeys.push('should'); // manually add due to should.js prototype pollution
+    require('../../test/assertions');
+  } catch(err) { /* skip devDependencies if unavailable */ }
+
+  console.log('Available vars:', contextKeys.sort().join(', '));
 
   const repl = require('repl').start({
     useGlobal: true, // enable should.js prototype pollution

